NuGet/Microsoft.ChakraCore.Debugger.nuspec

```diff
@@ -17,15 +17,15 @@
   </metadata>
 
   <files>
-    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="include" />
-    <file src="..\lib\Debugger.ProtocolHandler\ChakraDebugProtocolHandler.h" target="include" />
+    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="build\native\include" />
+    <file src="..\lib\Debugger.ProtocolHandler\ChakraDebugProtocolHandler.h" target="build\native\include" />
 
-    <file src="..\build\bin\x64\Debug\*" target="lib\x64\Debug" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
-    <file src="..\build\bin\x64\Release\*" target="lib\x64\Release" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
-    <file src="..\build\bin\x86\Debug\*" target="lib\x86\Debug" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
-    <file src="..\build\bin\x86\Release\*" target="lib\x86\Release" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
-    <file src="..\build\bin\ARM\Debug\*" target="lib\ARM\Debug" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
-    <file src="..\build\bin\ARM\Release\*" target="lib\ARM\Release" exclude="**\*.ilk;**\*ChakraCore.dll;**\*ChakraCore.pdb" />
+    <file src="..\build\bin\x64\Debug\ChakraCore.Debugger.???" target="lib\native\x64\Debug" exclude="**\*.ilk;**\*.exp" />
```

Comment:
>native [](start = 75, length = 6)

Similar to previous comment, why specify "native"? What is the recommended folder structure?

Reply:
There is no absolute standard for NuGet native packages, though there are different recommendations.

I'm adhering to the convention followed by ChakraCore itself.